cloudformation/apps/aem/consolidated/main-c.yaml

```diff
@@ -141,3 +141,7 @@ Resources:
             \ ${DataBucketNameParameter} ${MainStackPrefixParameter} author-publish-dispatcher\
             \ ${AemAwsStackProvisionerVersionParameter}\n"
     Type: AWS::EC2::Instance
+Outputs:
+    AppInstanceIP:
```

Comment:
Just to keep the terminology consistent with what the users are already familiar with, please rename `AppInstanceIp` into `AuthorPublishDispatcherInstanceIp` .